.github/workflows/phpunit.yml

```diff
@@ -3,13 +3,25 @@ name: PHPUnit-9
 on: [push, pull_request]
 
 jobs:
-  phpunit-php81:
+  phpunit-php82:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+      - name: Download phpunit
+        run: |
+          wget --no-check-certificate -O phpunit https://phar.phpunit.de/phpunit-9.phar
```

Comment:
remove `--no-check-certificate `

Reply:
It's Ok, i will remove it, but i know sometime on some linux distribute will raise this issue:

- https://serverfault.com/questions/314635/why-wget-doesnt-verify-ssl-certificates
- https://stackoverflow.com/questions/73400512/certificate-issues-with-centos7-with-curl

That why with ci proccess to forced download something i alway use option `--no-check-certificate`

Reply:
your systems may be broken, but If certs in GHA are broken it's a serious issue and you must contact github support. or in this particular case phar.phpunit.de owner if server cert is broken.

deal when the problem appears, not decrease security because "maybe".

it's also probably bad to pick moving target like version 9, not some pinned version like 9.1, but I haven't done research is there was way to install specific 9.x

Reply:
@glensc specific PHPUnit 9.5.26

Reply:
@hungtrinh your answer "specific PHPUnit 9.5.26" makes no sense. please form a full sentence.

Reply:
Mr @glensc I pick moving target to PHPUnit 9.5.26.  [Please review it](https://github.com/Shardj/zf1-future/pull/275/commits/168a6dfe7d80516ff45f9f9a136b252c0a4b84bf)